org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/resources/example_snippets/CallInvoker.xml

```diff
@@ -1,100 +1,100 @@
 <bpel:assign xmlns:bpel="http://docs.oasis-open.org/wsbpel/2.0/process/executable" name="AssignCreateEC2SIRequest" validate="no">
-	<bpel:copy>
-		<bpel:from>
-			<bpel:literal>
-				<impl:invokeOperationAsync xmlns:impl="http://siserver.org/schema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-					<impl:CsarID>TU_Wien_BIG_Test.csar</impl:CsarID>
+    <bpel:copy>
```

Comment:
In case you only changed spaces <-> tabs, please do not commit the change. It  makes it a lot easier for review when "real" changes have to be reviewed.

Furthermore, we have the 1000 LOC-rule at eclipse (see https://github.com/eclipse/winery/wiki/IP-process).